o1vm/src/mips/tests.rs

```diff
@@ -42,7 +42,7 @@ fn test_mips_number_constraints() {
     for instr in Instruction::iter().flat_map(|x| x.into_iter()) {
         match instr {
             RType(rtype) => match rtype {
-                JumpRegister | SyscallExitGroup | Sync => assert_num_constraints(&instr, 0),
+                JumpRegister | SyscallExitGroup | Sync => assert_num_constraints(&instr, 1),
```

Comment:
Note: the additional constraint seems to be the one for the instruction decomposition. :+1: